modules/dnn/src/layers/layers_common.cpp

```diff
@@ -195,7 +195,7 @@ void getConvolutionKernelParams(const LayerParams &params, std::vector<size_t>&
     util::getStrideAndPadding(params, pads_begin, pads_end, strides, padMode, kernel.size());
     util::getParameter(params, "dilation", "dilation", dilations, true, std::vector<size_t>(kernel.size(), 1));
     util::getParameter(params, "adj", "adj", adjust_pads, true, std::vector<size_t>(kernel.size(), 0));
-    useWinograd = params.get<bool>("use_winograd", true);
+    useWinograd = params.get<bool>("use_winograd", useWinograd);
```

Comment:
Hi @asmorkalov. Previously, due to the existence of this line of code, we turned on Winograd by default.